tests/examples/pacer/nda/ca2_3.json

```diff
@@ -15,7 +15,8 @@
           "pacer_case_id": "55129",
           "pacer_doc_id": "00208754210",
           "pacer_magic_num": "59a7f7615f78153b",
-          "pacer_seq_no": null
+          "pacer_seq_no": null,
+          "short_description": "Appellant/Petitioner Reply Brief FILED"
```

Comment:
The "FILED" here looks a little off. Is this right?

Reply:
Do you know if there is a place in PACER to confirm the entry short description for appellate cases? (Similar to the docket history reports available for district courts.)

In this case, the full description mentions the word "FILED".
![Screenshot 2023-04-21 at 16 51 24](https://user-images.githubusercontent.com/486004/233745811-e87a93e1-d8d0-499c-a417-05ac1800ebdb.png)

The subject from where we're taking the short description is the following:
![Screenshot 2023-04-21 at 16 44 05](https://user-images.githubusercontent.com/486004/233745855-614267bf-f5b1-4a7b-b2a8-28117bc7b615.png)

Also, in appellate notifications (NDAs) the document description is also in the footer and it matches the one in the subject.
![Screenshot 2023-04-21 at 16 44 17](https://user-images.githubusercontent.com/486004/233746010-89f3cbeb-e0ce-4d1c-ac25-d282bf100383.png)

Reply:
Huh. Seems good. I'm not sure where to find the short descriptions in appellate, no.